sssom/constants.py

```diff
@@ -177,3 +166,43 @@ class SchemaValidationType(str, Enum):
     SchemaValidationType.JsonSchema,
     SchemaValidationType.PrefixMapCompleteness,
 ]
+
+
+class SSSOMSchemaViewSingleton(object):
```

Comment:
I think this works but we have one too many layers here

SSSOMSchemaViewSingleton -> SSSOMSchemaView -> SchemaView

I think you could have just made SSSOMSchemaView a singleton

Reply:
You're right. I combined them.

Reply:
I think I know what you mean: So basically the above should be `return self.view.get_class("mapping").slots` , correct?